composer.json

```diff
@@ -1,10 +1,10 @@
 {
-    "name": "laravel-frontend-presets/zurb-foundation",
-    "description": "Laravel 5.5.x Front-end preset for Zurb Foundation",
+    "name": "crazyinventor/zurb-foundation-laravel-frontend-presets",
+    "description": "Laravel 6.0.x Front-end preset for Zurb Foundation",
     "keywords": ["laravel", "preset", "zurb-foundation", "foundation"],
     "license": "MIT",
     "require": {
-        "laravel/framework": "^5.5"
+        "laravel/framework": "^6.0"
```

Comment:
Can you make it so older versions are still supported?

Reply:
No idea if or how this can be done, sorry. I would just use the v2 tree for Laravel 5.x and v3 for Laravel 6.x. You can still provide updates for v2, if necessary.